src/KOKKOS/kokkos_type.h

```diff
@@ -1154,10 +1164,20 @@ typedef SNAComplex<SNAreal> SNAcomplex;
 #define ISFINITE(x) std::isfinite(x)
 #endif
 
-#define LAMMPS_LAMBDA KOKKOS_LAMBDA
```

Comment:
Why can we not use `KOKKOS_LAMBDA` here? If specialization is needed for `SYCL`, then shouldn't we add this to `KOKKOS_LAMBDA` inside Kokkos instead of in LAMMPS?

Reply:
Yes, you're right. Looks like a rebasing artifact. Fixed.